handler.py

```diff
@@ -9,13 +9,19 @@
 from util.response import Response
 from scanners.http_observatory_scanner import HTTPObservatoryScanner
 from util.randomizer import Randomizer
+from scanners.ctlog_scanner import CTLogScanner
```

Comment:
I'm conflicted on whether this should be a "scanner", when technically, it would function more like a host aggregator of sorts.  I wonder if we want to have a brainstorm on what we would call this?

Reply:
This is a slightly different model than we've used before, where previously we were providing the hostlist, this discovers hosts.